{"time":"2025-03-15T16:53:22.9408114+05:30","level":"INFO","msg":"stream: starting","core version":"0.19.8","symlink path":"C:\\Users\\singh\\OneDrive\\Documents\\GitHub\\DA6401_Assignment-01\\wandb\\run-20250315_165322-63rh289e\\logs\\debug-core.log"}
{"time":"2025-03-15T16:53:23.0604911+05:30","level":"INFO","msg":"created new stream","id":"63rh289e"}
{"time":"2025-03-15T16:53:23.0604911+05:30","level":"INFO","msg":"stream: started","id":"63rh289e"}
{"time":"2025-03-15T16:53:23.0604911+05:30","level":"INFO","msg":"handler: started","stream_id":"63rh289e"}
{"time":"2025-03-15T16:53:23.0604911+05:30","level":"INFO","msg":"sender: started","stream_id":"63rh289e"}
{"time":"2025-03-15T16:53:23.0604911+05:30","level":"INFO","msg":"writer: Do: started","stream_id":"63rh289e"}
{"time":"2025-03-15T16:53:24.0617828+05:30","level":"INFO","msg":"Starting system monitor"}
{"time":"2025-03-15T16:56:19.8629336+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/singhsonalika5-indian-institute-of-technology-madras/Fashion_MNIST_Images/63rh289e/file_stream\": read tcp 192.168.25.107:56999->35.186.228.49:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-15T16:56:33.3572193+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/singhsonalika5-indian-institute-of-technology-madras/Fashion_MNIST_Images/63rh289e/file_stream\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-03-15T16:56:58.786629+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/singhsonalika5-indian-institute-of-technology-madras/Fashion_MNIST_Images/63rh289e/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-15T16:57:28.3075137+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/singhsonalika5-indian-institute-of-technology-madras/Fashion_MNIST_Images/63rh289e/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-15T16:57:39.4740321+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": context deadline exceeded"}
{"time":"2025-03-15T16:58:07.6007845+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/singhsonalika5-indian-institute-of-technology-madras/Fashion_MNIST_Images/63rh289e/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-15T16:58:11.9293584+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": net/http: request canceled (Client.Timeout exceeded while awaiting headers)"}
{"time":"2025-03-15T16:58:46.2001484+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": net/http: request canceled (Client.Timeout exceeded while awaiting headers)"}
{"time":"2025-03-15T16:59:04.2613591+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/singhsonalika5-indian-institute-of-technology-madras/Fashion_MNIST_Images/63rh289e/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-15T17:00:25.3323177+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/singhsonalika5-indian-institute-of-technology-madras/Fashion_MNIST_Images/63rh289e/file_stream\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-15T17:00:32.3299683+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": read tcp 192.168.25.107:56768->35.186.228.49:443: wsarecv: An established connection was aborted by the software in your host machine."}
{"time":"2025-03-15T17:00:34.5401288+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp: lookup api.wandb.ai: no such host"}
{"time":"2025-03-15T17:01:00.5099231+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-15T17:01:31.224941+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp 35.186.228.49:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2025-03-15T17:01:45.1017412+05:30","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/singhsonalika5-indian-institute-of-technology-madras/Fashion_MNIST_Images/63rh289e/file_stream\": dial tcp 35.186.228.49:443: connectex: An established connection was aborted by the software in your host machine."}
{"time":"2025-03-15T17:08:38.6203104+05:30","level":"INFO","msg":"Stopping system monitor"}
{"time":"2025-03-15T17:08:38.6220732+05:30","level":"INFO","msg":"Stopped system monitor"}
{"time":"2025-03-15T17:08:39.6352999+05:30","level":"INFO","msg":"handler: operation stats","stats":{"operations":[{"desc":"uploading output.log","runtime_seconds":0.3228333,"progress":"658B/658B"},{"desc":"uploading wandb-summary.json","runtime_seconds":0.3228333,"progress":"212B/212B"},{"desc":"uploading config.yaml","runtime_seconds":0.0345559,"progress":"962B/962B"}],"total_operations":3}}
{"time":"2025-03-15T17:08:39.9401226+05:30","level":"INFO","msg":"fileTransfer: Close: file transfer manager closed"}
{"time":"2025-03-15T17:08:40.2570193+05:30","level":"INFO","msg":"stream: closing","id":"63rh289e"}
{"time":"2025-03-15T17:08:40.2570193+05:30","level":"INFO","msg":"handler: closed","stream_id":"63rh289e"}
{"time":"2025-03-15T17:08:40.2570193+05:30","level":"INFO","msg":"writer: Close: closed","stream_id":"63rh289e"}
{"time":"2025-03-15T17:08:40.2570193+05:30","level":"INFO","msg":"sender: closed","stream_id":"63rh289e"}
{"time":"2025-03-15T17:08:40.2580974+05:30","level":"INFO","msg":"stream: closed","id":"63rh289e"}
